src/consensus/aft/raft.h

```diff
@@ -949,8 +949,6 @@ namespace aft
 
       const auto prev_term = get_term_internal(prev_idx);
       const auto term_of_idx = get_term_internal(end_idx);
-      const bool contains_new_view =
-        (state->new_view_idx > prev_idx) && (state->new_view_idx <= end_idx);
```

Comment:
Ok, I'm sneaking in "don't even populate `contains_new_view`" in this change too. Because we never use the read value, and haven't for a long time, I claim this is fine.